README.md

```diff
@@ -120,15 +120,90 @@ $D$(in) & $P_u$(lbs) & $u_u$(in) & $\beta$ & $G_f$(psi.in)\\
 
 ## 编译
 
+> - 以下编译操作都**需要用户进入 `tjumain.tex` 所在目录下**
+> - 有多种编译形式，本质都一样，选择某种自己喜欢的即可，欢迎补充。
+
+### 使用`Latexmk`编译
+
+#### 首先检查是否安装了`Latexmk`
```

Comment:
Checking installation and explaning working mechanism don't seem to be a template's scope. We should only provide minmal and proper commands to unblock users from using the template. These parts would be fit for a latex 101 guide for TJU students